modules/custom_role_iam/variables.tf

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "org_id" {
+  type        = string
+  description = "Organization ID"
+}
+
+variable "project_id" {
+  type        = string
+  description = "Project ID"
+}
+
+variable "title" {
+  type        = string
+  description = "Title of the Custom Role."
+}
+
+variable "role_id" {
+  type        = string
+  description = "ID of the Custom Role."
+}
+
+variable "permissions" {
+  type        = list(string)
+  description = "IAM permissions assigned to Custom Role."
+}
+
+variable "description" {
```

Comment:
Description should be optional, maybe we can make a sane default for this? WDYT

Reply:
Done.

Reply:
It looks like this variable isn't used anywhere?

Reply:
description is optional in the resource. Should I remove the variable if its optional?

Reply:
The issue is we seemingly don't set it on the resource at all - var.description isn't referenced anywhere.

We can leave it as optional, with an empty (null) default.

Reply:
Sure, done.